src/Orleans.Core/Diagnostics/MessagingTrace.cs

```diff
@@ -7,7 +7,7 @@ namespace Orleans.Runtime
 {
     internal class MessagingTrace : DiagnosticListener, ILogger
     {
-        public const string Category = "Microsoft.Orleans.Messaging";
+        public const string Category = "Orleans.Messaging";
```

Comment:
Would it be better to split category renaming into a separate PR? I'm concerned that people might have taken dependencies on those categories in their monitoring/alerting configurations.

Reply:
Makes sense - I assume the implication is that we'd save the rename PR for 3.2 or 4.0, rather than 3.1.x

Reply:
Yes, we'll merge it to `master` , just won't cherry-pick for 3.1.4.

Reply:
Fixed